app/algorithms/normal.ts

```diff
@@ -0,0 +1,110 @@
+import isEmpty = require('lodash/isEmpty');
+import { Status, Vote, Alternative, ElectionResult } from './types';
+
+// This is a TypeScript file in a JavaScript project so it must be complied
+// If you make changes to this file it must be recomplied using `tsc` in
+// order for the changes to be reflected in the rest of the program.
+//
+// app/models/election .elect() is the only file that uses this function
+// and importes it from normal.js, which is the compiled result of this file.
+//
+//
+
+export type Count = { [key: string]: number };
```

Comment:
Any reason this is not in the types file? Also, to make this even better, since there is always a `blank` key, it would be even better to include `blank` specifically as a key

Reply:
I can factor out this Count into `types.ts`, but then it will be used by both STV and Norma (which is fine), but then blank can't be a a key.